### PR TITLE
[libgpod] only linux

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -532,7 +532,6 @@ libcopp:arm64-windows=fail
 libcopp:arm-uwp=fail
 # Missing system libraries on linux to run/prepare autoconf
 libgpod:x64-linux=fail
```
Comment (LilyWangLL) [inline scripts/ci.baseline.txt:534]:
```suggestion
```
`libgpod` supports Linux, this code needs be removed. But `libgpod` build failed with the following error now, could you please check and fix?
```
Building package libgpod[core]:x64-linux...
-- Downloading https://github.com/fadingred/libgpod/archive/4a8a33ef4bc58eee1baca6793618365f75a5c3fa.tar.gz -> fadingred-libgpod-4a8a33ef4bc58eee1baca6793618365f75a5c3fa.tar.gz...
-- Extracting source /home/vlilywang/Lily/0117/vcpkg/downloads/fadingred-libgpod-4a8a33ef4bc58eee1baca6793618365f75a5c3fa.tar.gz
-- Applying patch configure.ac.patch
-- Using source at /home/vlilywang/Lily/0117/vcpkg/buildtrees/libgpod/src/5f75a5c3fa-9e39aa1f67.clean
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: intltoolize --force --copy --automake
    Working Directory: /home/vlilywang/Lily/0117/vcpkg/buildtrees/libgpod/src/5f75a5c3fa-9e39aa1f67.clean
    Error code: No such file or directory
    See logs for more information:

Call Stack (most recent call first):
  ports/libgpod/portfile.cmake:12 (vcpkg_execute_required_process)
  scripts/ports.cmake:142 (include)
```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: scripts/ci.baseline.txt (hunk 1 above)
Comment (autoantwort) [reply scripts/ci.baseline.txt:534]: Hm I have no linux to test this currently

Comment (autoantwort) [reply scripts/ci.baseline.txt:534]: I also have not changed anything

**Comment by autoantwort** [comment]
This change was done by another PR

**Pull request closed without merging**